projects/js-packages/charts/src/components/pie-semi-circle-chart/stories/index.docs.mdx

```diff
@@ -0,0 +1,458 @@
+		label="Single Category"
+	/>` }
+/>
+
+## Advanced Features
+
+### Value Display Formatting
+
+Use the `valueDisplay` property to show formatted values in tooltips and legends:
+
+<Source
+	language="jsx"
+	code={ `const formattedData = [
+		{
+			label: 'Users',
+			value: 15000,
+			valueDisplay: '15K users', // Custom formatted display
+			percentage: 60,
+		},
+		{
+			label: 'Revenue',
+			value: 10000,
+			valueDisplay: '$10K', // Currency formatting
+			percentage: 40,
+		},
+	];
+
+	<PieSemiCircleChart
+		data={ formattedData }
+		withTooltips={ true }
+		showLegend={ true }
+	/>` }
+/>
+
+### Chart Integration
+
+The chart integrates with the chart context system for consistent behavior across chart types:
+
+- Automatic chart ID generation for accessibility
+- Theme consistency across multiple charts
+- Legend item registration for cross-chart interactions
+
+## Accessibility
+
+### Screen Reader Support
+
+The chart provides comprehensive screen reader support:
+- ARIA labels describe the chart purpose and data
+- Individual segments are announced with labels and values
+- Error states are clearly communicated
+- Legend items are properly associated with chart segments
+
+## Browser Compatibility
+
+### Modern Browser Support
+
+The chart uses modern SVG and CSS features with broad browser support:
+- **Chrome/Edge**: Full support for all features
+- **Firefox**: Complete functionality including animations
+- **Safari**: Full support with minor rendering optimizations
+
+### Performance Considerations
+
+- **Rendering**: Optimized SVG rendering for smooth interactions
+- **Memory**: Efficient re-rendering with React hooks optimization
+- **Animations**: CSS-based transitions for hover states
+
+## API Reference
+
+### PieSemiCircleChart
+
+Main component for rendering semi-circular pie charts.
+
+**Props:**
+
+| Prop | Type | Default | Description |
+| ---- | ---- | ------- | ----------- |
+| `data` | `DataPointPercentage[]` | - | **Required.** Array of data points to display |
+| `width` | `number` | `400` | Width of the chart in pixels (height is automatically half of width) |
+| `thickness` | `number` | `0.4` | Thickness of the pie segments (0-1, where 1 is full thickness) |
+| `clockwise` | `boolean` | `true` | Direction of segment rendering |
+| `label` | `string` | - | Text displayed above the chart |
+| `note` | `string` | - | Additional text displayed below the label |
+| `withTooltips` | `boolean` | `false` | Enable interactive tooltips on hover |
+| `showLegend` | `boolean` | `false` | Display legend below the chart |
+| `legendOrientation` | `'horizontal' \| 'vertical'` | `'horizontal'` | Legend layout direction |
+| `legendAlignmentHorizontal` | `'left' \| 'center' \| 'right'` | `'center'` | Horizontal legend alignment |
+| `legendAlignmentVertical` | `'top' \| 'bottom'` | `'bottom'` | Vertical legend position |
+| `legendShape` | `'circle' \| 'square' \| 'line'` | `'circle'` | Shape of legend indicators |
```

Comment:
Think this is supposed to be `'rect' | 'circle' | 'line'`

Reply:
Addressed in 69199786ac